checklists/alz_checklist.en.json

```diff
@@ -406,25 +406,13 @@
       "training": "https://learn.microsoft.com/learn/modules/azure-architecture-fundamentals/",
       "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups"
     },
-    {
```

Comment:
@tsc-buddy I had a question on this - to confirm, we are wanting to remove the recommendation to do tagging through policy?

Reply:
Hey @brsteph - great question, the feedback we received was that it was a double to the policy with the following GUID: e979377b-cdb3-4751-ab2a-b13ada6e55d7

 "text": "Enforce or appended resource tags through Azure Policy",
&
"text": "Identify required Azure tags and use the 'append' policy mode to enforce usage."

Reply:
Have tweaked the wording to reference azure policy on the remaining item. Let us know if you are happy with that?

Reply:
Looks good, thanks for the clarification!